CODEOWNERS

```diff
@@ -137,10 +137,10 @@ extensions/filters/common/original_src @klarose @mattklein123
 /*/extensions/grpc_credentials/example @wozz @yanavlasov
 /*/extensions/grpc_credentials/file_based_metadata @wozz @yanavlasov
 /*/extensions/internal_redirect @botengyao @penguingao
-/*/extensions/stat_sinks/dog_statsd @taiki45 @jmarantz
+/*/extensions/stat_sinks/dog_statsd @taiki45 @paul-r-gall
 /*/extensions/stat_sinks/graphite_statsd @vaccarium @mattklein123
-/*/extensions/stat_sinks/hystrix @trabetti @jmarantz
-/*/extensions/stat_sinks/metrics_service @ramaraochavali @jmarantz
+/*/extensions/stat_sinks/hystrix @trabetti @paul-r-gall
+/*/extensions/stat_sinks/metrics_service @ramaraochavali @paul-r-gall
```

Comment:
fwiw, I think you can still be on this list by just adding new person if you want.